sparkle/src/components/ContextItem.tsx

```diff
@@ -28,7 +30,15 @@ export function ContextItem({
         "s-flex s-w-full s-flex-col"
       )}
     >
-      <div className="s-flex s-flex-row s-gap-3 s-px-2 s-py-4">
+      <div
+        className={classNames(
+          "s-flex s-flex-row s-items-center s-gap-3 s-px-4 s-py-2",
+          onClick
+            ? "s-cursor-pointer s-transition s-duration-200 hover:s-bg-structure-50 active:s-bg-structure-100"
+            : ""
+        )}
+        onClick={onClick} // attach the onClick handler here
```

Comment:
com can be removed (is it our friend gpt4? :smile: )

Reply:
Totally is